test/test_memcached_store.rb

```diff
@@ -617,11 +682,27 @@ def with_read_only(client)
     client.read_only = previous
   end
 
+  def raises_when_not_swallowing_exceptions
```

Comment:
I think `assert_raises_when_not_swallowing_exceptions` would be a tiny bit clearer (add an `assert_` prefix to the method name).